lib/impure/db_sqlite.nim

```diff
@@ -183,13 +183,14 @@ proc close*(db: TDbConn) {.tags: [FDb].} =
   ## closes the database connection.
   if sqlite3.close(db) != SQLITE_OK: dbError(db)
     
-proc open*(connection, user, password, database: string): TDbConn {.
-  tags: [FDb].} =
+proc open*(connection, user, password, database: string, 
+           charset: string = "UTF-8"): TDbConn {.tags: [FDb].} =
   ## opens a database connection. Raises `EDb` if the connection could not
   ## be established. Only the ``connection`` parameter is used for ``sqlite``.
   var db: TDbConn
   if sqlite3.open(connection, db) == SQLITE_OK:
     result = db
+    exec(result, sql"PRAGMA encoding = ?", [charset])
```

Comment:
What if I want to leave the encoding as it is? There are many use cases to treat strings just as blobs and I don't want the database to do any encoding translations for me. So I think ``open`` should stay as it is, but instead a separate ``setEncoding`` proc should be added.